README.md

```diff
@@ -193,6 +193,29 @@ pip install s3fs
 RayTaskRunner(address="ray://1.23.199.255:10001")
 ```
 
+## Specifying resources requirements
+
+The `resources` context can be used to control the task’s resource requirements.
+
+For example, we can set the number of CPUs and GPUs to use for the `process` task:
+
+```python
+from prefect import flow, task
+from prefect_ray.task_runners import RayTaskRunner
+from prefect_ray.context import resources
+
+@task
+def process(x):
+    return x + 1
+
+
+@flow(task_runner=RayTaskRunner())
+def my_flow():
+    # equivalent to setting @ray.remote(num_cpus=4, num_gpus=2)
+    with resources(num_cpus=4, num_gpus=2):
```

Comment:
I would name this `remote_options` instead of `resources`. It seems useful to pass other `remote` parameters.